index.d.ts

```diff
@@ -467,6 +469,5 @@ declare namespace SendGrid {
     }
 }
 
-declare const mail: SendGrid.Helpers.Mail.Helper;
 declare const sendGrid: SendGrid.SendGridConstructor;
-export {mail, sendGrid as SendGrid};
\ No newline at end of file
+export = sendGrid;
```

Comment:
It is certainly a breaking change when it comes to the definition, however it corrects the definition to match the actual behaviour of the SendGrid JS library.

I've updated the interface above to include the rest of the library's exports as expected.